x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/details/execution_log_table/execution_log_table.tsx

```diff
@@ -0,0 +1,325 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SortOrder } from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
+import { DurationRange } from '@elastic/eui/src/components/date_picker/types';
+import { get } from 'lodash';
+import styled from 'styled-components';
+import React, { useCallback, useMemo, useState } from 'react';
+import {
+  EuiTextColor,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiSuperDatePicker,
+  OnTimeChangeProps,
+  OnRefreshProps,
+  OnRefreshChangeProps,
+  EuiSpacer,
+  EuiSwitch,
+  EuiBasicTable,
+} from '@elastic/eui';
+import { buildFilter, FILTERS } from '@kbn/es-query';
+import { MAX_EXECUTION_EVENTS_DISPLAYED } from '@kbn/securitysolution-rules';
+import { RULE_DETAILS_EXECUTION_LOG_TABLE_SHOW_METRIC_COLUMNS_STORAGE_KEY } from '../../../../../../../common/constants';
+import { AggregateRuleExecutionEvent } from '../../../../../../../common/detection_engine/schemas/common';
+
+import {
+  UtilityBar,
+  UtilityBarGroup,
+  UtilityBarSection,
+  UtilityBarText,
+} from '../../../../../../common/components/utility_bar';
+import { useSourcererDataView } from '../../../../../../common/containers/sourcerer';
+import { useAppToasts } from '../../../../../../common/hooks/use_app_toasts';
+import { useKibana } from '../../../../../../common/lib/kibana';
+import { SourcererScopeName } from '../../../../../../common/store/sourcerer/model';
+import { useRuleExecutionEvents } from '../../../../../containers/detection_engine/rules';
+import * as i18n from './translations';
+import { EXECUTION_LOG_COLUMNS, GET_EXECUTION_LOG_METRICS_COLUMNS } from './execution_log_columns';
+import { ExecutionLogSearchBar } from './execution_log_search_bar';
+
+const EXECUTION_UUID_FIELD_NAME = 'kibana.alert.rule.execution.uuid';
```

Comment:
May be able to reference from event-log/common, but if field names aren't accessible either update or pull these out to their own package for re-use.

Reply:
Keeping as-is for now, but still think we should publish from event-log/common.